src/privacy-policy.md.njk

```diff
@@ -5,52 +5,105 @@ layout: layout-single-page-prose.njk
+- that the processing of your personal data is restricted in certain circumstances - for example, where accuracy is contested
+
+If you gave your consent for us to collect and process your data, you have the right to:
+- withdraw your consent — this can be done at any time
+- request a copy of your personal data — this copy will be provided in a structured, commonly used and machine-readable format
+
+## Questions and complaints
 
-## Links to other websites
+Contact the GDS Privacy Office if you:
+- have any questions about anything in this document
+- think that your personal data has been misused or mishandled
+- want to make a [subject access request (SAR)](https://ico.org.uk/your-data-matters/your-right-of-access/)
 
-The GOV.UK Design System contains links to other websites.
+The contact details for the data controller are: Cabinet Office (Government Digital Service), The White Chapel Building, 10 Whitechapel High Street, London, E1 8QS, or [gds-privacy-office@digital.cabinet-office.gov.uk](mailto:gds-privacy-office@digital.cabinet-office.gov.uk).
 
-This privacy policy only applies to the GOV.UK Design System, and does not cover other government services and transactions that we link to. These services have their own terms and conditions and privacy policies.
+The contact details for the data controller’s Data Protection Officer are: Stephen Jones, Data Protection Officer, Cabinet Office, 70 Whitehall, London, SW1A 2AS, or [dpo@cabinetoffice.gov.uk](mailto:dpo@cabinetoffice.gov.uk).
 
-### Following a link to another website
+The Data Protection Officer provides independent advice and monitoring of Cabinet Office’s use of personal information.
 
-If you go to another website from this one, read the privacy policy on that website to find out what it does with your information.
+If you consider that your personal data has been misused or mishandled, you may make a complaint to the Information Commissioner, who is an independent regulator. The Information Commissioner can be contacted at: Information Commissioner's Office, Wycliffe House, Water Lane, Wilmslow, Cheshire, SK9 5AF, or 0303 123 1113, or [casework@ico.org.uk](mailto:casework@ico.org.uk). 
 
-### Following a link to the GOV.UK Design System from another website
+Any complaint to the Information Commissioner is without prejudice to your right to seek redress through the courts.
 
-If you come to the GOV.UK Design System from another website, we may receive information from the other website. We do not use this data. You should read the privacy policy of the website you came from to find out more about this.
+## Changes to this notice
+We may change this privacy notice. When we make changes to this notice, the ‘last updated’ date at the top of this page will also change. Any changes to this privacy notice will apply to you and your data immediately. If these changes affect how your personal data is processed, GDS will take reasonable steps to make sure you know.
```

Comment:
Currently there is no 'last updated' date, is there a ticket to add that separately?

Reply:
Nope, obviously this was carried over from elsewhere. I'll add to the bottom manually (so it's similar to [what's on gov.uk](https://www.gov.uk/help/privacy-notice)) and ask the team if we want to style this differently than default text.

```suggestion
We may change this privacy notice. When we make changes to this notice, the ‘last updated’ date at the bottom of this page will also change. Any changes to this privacy notice will apply to you and your data immediately. If these changes affect how your personal data is processed, GDS will take reasonable steps to make sure you know.

Last updated 19 May 2021
```